navigator/backend/scenarios/rental/ui-backend.conf

```diff
@@ -0,0 +1,14 @@
+users {
```

Comment:
This file is no longer necessary, Navigator should pick up the parties from `daml.yaml`.

Reply:
I tried running it but it threw an exception because it could not find this file (I'm on the latest master to the best of my knowledge). Do you know what the cause may be?

Reply:
Since this is part of a manual procedure and breaks nothing I'll remove the file as suggested, but you may want to keep an eye on this.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/1950/commits/4d5451ff637630f6f5012e8e4d401140af578f71

Reply:
As suspected, the [documented procedure](https://github.com/digital-asset/daml/pull/1950/files#diff-8f0c8b5cfd0b3b1b6d237ffd59aebbae) doesn't work without this file. It fails with the following error:
```
No configuration file found!
Please specify a configuration file and restart Navigator.
Config file path was 'ui-backend.conf'.
Hint: use the create-config command to create a sample config file.
Exception in thread "main" java.lang.RuntimeException: No configuration file found.
	at scala.sys.package$.error(package.scala:27)
	at com.digitalasset.navigator.UIBackend.run(UIBackend.scala:301)
	at com.digitalasset.navigator.UIBackend.$anonfun$main$1(UIBackend.scala:279)
	at com.digitalasset.navigator.UIBackend.$anonfun$main$1$adapted(UIBackend.scala:279)
	at scala.Option.foreach(Option.scala:257)
	at com.digitalasset.navigator.UIBackend.main(UIBackend.scala:279)
	at com.digitalasset.navigator.NavigatorBackend.main(NavigatorBackend.scala)
```